bubble_sort/bubble_sort.py

```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort
+ 
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -1,5 +1,6 @@
 # Python program for implementation of Bubble Sort
- 
+
+
 def bubbleSort(arr):
     n = len(arr)
  
```